.github/pull_request_template.md

```diff
@@ -1,23 +1,23 @@
-**Thank you for submitting this pull request**
+**JIRA**: [link](https://www.example.com) 
 
-**JIRA**: _(please edit the JIRA link if it exists)_ 
+**Referenced Pull Requests** (if any):
 
-[link](https://www.example.com)
```

Comment:
Referenced why? Again, you and I know that, but does the target audience?
I understand the intention to keep it short. But if that's the intention, we might as well just not include this section. The outsiders won't be doing cross-repo PRs.